services/director/src/simcore_service_director/registry_proxy.py

```diff
@@ -65,17 +116,22 @@ class ServiceType(enum.Enum):
     raise exceptions.RegistryConnectionError("Unknown registry authentification type: {}".format(url))
 
 async def _registry_request(path: URL, method: str ="GET") -> Tuple[Dict, Dict]:
+    cache_key = "{}_{}".format(path, method)
```

Comment:
I wonder why you did not create this with a decorator
Why you only use it here and not in ``_auth_registry_request`` as well?

Reply:
never did a decorator before, that is probably why.
_auth_registry_request is a subfunction that gets called by registry_request when authentication is necessary, therefore no need to cache so deep.